### PR TITLE
Update medicine-and-science-in-sports-and-exercise.csl

### PR DESCRIPTION
**Pull request by POBrien333**
https://forums.zotero.org/discussion/comment/295191/#Comment_295191

### CODE REVIEW

**Comment by POBrien333** [comment]
Actually. This needs a bit more work. Some conditionals for url and doi for journals. only when there is no pages.

**Comment by POBrien333** [comment]
Could you both check the changes I made in macro="access" to match the guidelines?

File: medicine-and-science-in-sports-and-exercise.csl [hunk 1]
```diff
@@ -66,27 +71,58 @@
...
-          <date-part name="day" prefix=" "/>
-        </date>
-      </group>
-      <group>
-        <text value="Available from: "/>
-        <text variable="URL"/>
-      </group>
-    </group>
+    <choose>
+      <if type="article-journal" match="any">
```
Comment (adam3smith) [inline medicine-and-science-in-sports-and-exercise.csl]: I'm afraid this doesn't quite work: If you look at
Vickers AJ. Time course of muscle soreness following different types of exercise. BMC Musculoskeletal Disorders [Internet]. 2001 [cited 2001 May 31];2(5). Available from: http://www.biomedcentral.com/1471-2474/2/5. doi:10.1186/1471-2474-2-5.

The volume and issue information is between access date and URL/DOI. The way you'd have to do this is to the [cited 2001 May 3] onto the date macro and then have only URL/DOI in access all the way at the end.

**Comment by POBrien333** [comment]
thanks. That made sense. This should work.

**Comment by adam3smith** [comment]
perfect, thanks!

**Pull request merged**